args/core/math/glm/detail/type_vec3.inl

```diff
@@ -165,26 +178,26 @@ namespace args::core::math
 	// -- Component accesses --
 
 	template<typename T, qualifier Q>
-	GLM_FUNC_QUALIFIER GLM_CONSTEXPR T & vec<3, T, Q>::operator[](typename vec<3, T, Q>::length_type i)
+	GLM_FUNC_QUALIFIER GLM_CONSTEXPR T& vec<3, T, Q>::operator[](typename vec<3, T, Q>::length_type i)
 	{
 		assert(i >= 0 && i < this->length());
-		switch(i)
+		switch (i)
 		{
 		default:
-			case 0:
-		return x;
-			case 1:
-		return y;
-			case 2:
-		return z;
+		case 0:
+			return x;
+		case 1:
+			return y;
+		case 2:
+			return z;
 		}
```

Comment:
you could also add an operator for char types to allow for this syntax: `v['x']` which would be useful if we want to expose this operator to a scripting language

Reply:
would be usefull but this is just default glm